packages/core/src/state-management/__tests__/anchor-resuming.test.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, jest } from '@jest/globals'
```

Comment:
ugh sorry, I just added some tests to `anchor-resuming-service.test.ts` as part of the fixes I just put in.  LMK if you want any help porting them over, but we should make sure not to lose the coverage

Reply:
Please! If you could port the changes, that would be great!

Reply:
Done!